status-monitor.lic

```diff
@@ -22,15 +22,23 @@ class GameFilter
 
   def initialize
     @seen_cache_name = "seen_messages_#{checkname}.dat"
+    @seen_backup_name = "backup/seen_messages_#{checkname}.bak"
```

Comment:
💡 Why store this in a subdirectory?

Reply:
The entire reason is that I saw my lich install had a lich\backup directory with nothing in it and I thought, "Huh, maybe this was supposed to be something once upon a time.  Guess I'll put the backups in here."

Unless that was something _I_ made and forgot about.  More than possible.